pyprland/plugins/scratchpads/__init__.py

```diff
@@ -557,6 +548,16 @@ async def run_show(self, uid: str) -> None:
         assert monitor
         assert scratch.full_address, "No address !"
 
+        excluded_ids = scratch.conf.get("excludes", [])
+        excludes_stash = scratch.conf.get("excludes_stash", False)
+        if excluded_ids == "*":
+            excluded_ids = [excluded.uid for excluded in self.scratches.values() if excluded.uid != scratch.uid]
+        for e_uid in excluded_ids:
+            excluded = self.scratches.get(e_uid)
```

Comment:
why did you get rid of the assertion?

Reply:
Sorry, I thought it was my own for debugging.